test/e2e-go/cli/algoh/expect/algohExpectCommon.exp

```diff
@@ -55,9 +65,11 @@ proc ::Algoh::CreateNetwork { NETWORK_NAME NETWORK_TEMPLATE TEST_ROOT_DIR } {
 }
 
 proc ::Algoh::StartNode { TEST_ALGO_DIR } {
+    set ::GLOBAL_TEST_ALGO_DIR $TEST_ALGO_DIR
 
     puts "Primary node start"
-    # Start node
+    set timeout 15
```

Comment:
I'm not a big fan of extending timeouts without understanding of why we're doing so.
It's true the running on certain platform is substantially slower, but I'd like to understand which portion of the startup is slowing us down and address that directly ( or, knowingly, avoid addressing that.. )

Reply:
also, having timeouts per-platform might be a good thing to do here.

Reply:
I agree, it possibly requires additional research. I can remove timeout but other fixes is good to have anyway.

Reply:
other changes are fine.

Reply:
removed